x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -13273,19 +13273,20 @@
     "xpack.enterpriseSearch.content.index.pipelines.settings.reduceWhitespaceDescription": "Supprimer automatiquement l'espace supplémentaire de vos documents",
     "xpack.enterpriseSearch.content.index.pipelines.settings.reduceWhitespaceLabel": "Réduire l'espace",
     "xpack.enterpriseSearch.content.index.pipelines.settings.runMlInferenceDescrition": "Améliorer vos données à l'aide de modèles de ML entraînés compatibles",
+    "xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.body": "ELSER (Elastic Learned Sparse EncodeR) est notre modèle d'apprentissage automatique pré-entrainé conçu pour utiliser efficacement le contexte dans les requêtes en langage naturel, dont la version 2 est désormais disponible. Ce modèle donne de meilleurs résultats que le BM25 sans entrainement supplémentaire sur vos données.",
```

Comment:
Please revert the translation changes. We should not be touching those, they'll be marked as changed and automatically processed and re-translated by our translation service.

Reply:
@sphilipse I was not aware of the existence of the translation service!!! I will revert the changes.